sda/local.go

```diff
@@ -398,6 +398,11 @@ func (l *LocalTest) NewLocalConode(port int) *Conode {
 
 }
 
+// NewContext creates a context from a ServerIdentity.
+func (l *LocalTest) NewContext(si *network.ServerIdentity, servID ServiceID, manager *serviceManager) *Context {
```

Comment:
mhhh I'm not a fan of providing Yet Another public method to `LocalTest`. Would it be not possible to have `func (l *LocalTest) GetContext(c *sda.Conode,servID ServiceID)` ? 
A Context without its service is useless.